packages/storybook/src/components/gallery/Gallery.tsx

```diff
@@ -28,9 +28,8 @@ const Gallery = ({ components }: Props) => {
             <OdsCard className={ styles['gallery__items__item'] }
                      color={ ODS_CARD_COLOR.neutral }
                      key={ idx }>
-              <Canvas className={ styles['gallery__items__item__preview'] }
-                      of={ component.story }
-                      sourceState="none" />
+              <CanvasWithoutContext className={ styles['gallery__items__item__preview'] }
```

Comment:
breaks the rendering style
<img width="820" alt="Capture d’écran 2025-01-10 à 11 17 35" src="https://github.com/user-attachments/assets/e846de6b-61e8-477c-8859-c0a7577a991a" />